api/dao/reaperutil.py

```diff
@@ -147,8 +148,12 @@ def create_container_hierarchy(metadata):
 
     if acquisition.get('timestamp'):
         acquisition['timestamp'] = dateutil.parser.parse(acquisition['timestamp'])
-        config.db.projects.update_one({'_id': project_obj['_id']}, {'$max': dict(timestamp=acquisition['timestamp']), '$set': dict(timezone=acquisition.get('timezone'))})
-        config.db.sessions.update_one({'_id': session_obj['_id']}, {'$min': dict(timestamp=acquisition['timestamp']), '$set': dict(timezone=acquisition.get('timezone'))})
+        #FIXME timezone is always set to the value of the new acquisition
+        config.db.projects.update_one({'_id': project_obj['_id']}, {'$max': dict(timestamp=acquisition['timestamp'])})
+        session_operations = {'$min': dict(timestamp=acquisition['timestamp'])}
+        if acquisition.get('timezone'):
+            session_operations['$set'] = {'timezone': acquisition['timezone']}
+        config.db.sessions.update_one({'_id': session_obj['_id']}, session_operations)
```

Comment:
I don't see the need for the FIXME here. That behavior is fine and expected.

Also, I thought we decided that the logic on 152 would be removed. We simply don't have timestamps on projects.